Tests/MeshLib/TestRasterToMesh.cpp

```diff
@@ -270,26 +271,25 @@ TEST_F(RasterToMeshTest, convertRasterToQuadMeshNone)
 #ifdef OGS_BUILD_GUI
 TEST_F(RasterToMeshTest, vtkImage)
 {
-    double x0;
-    double y0;
-    double spacing;
-    vtkImageAlgorithm* raster = VtkRaster::loadImage(_file_name);
+    double const spacing = std::numeric_limits<double>::quite_NaN();
```

Comment:
Is the correct header included for `quite_NaN`?

Reply:
There are uses of epsilon somewhere else in that file...
But those, who are mighty of proper spelling, don't get errors... :)

Reply:
```suggestion
    double const spacing = std::numeric_limits<double>::quiet_NaN();
```